src/main/java/Duke.java

```diff
@@ -1,10 +1,111 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+    public static Boolean test_existence(Task[] tasks_list, String instruction){
```

Comment:
variable name is not that clear needs to read as a true false name(e.g. test_existence should be doesItExist)
also consider making the name should be in camel case